packages/koa-shopify-graphql-proxy/README.md

```diff
@@ -85,3 +85,27 @@ app.use(
   }),
 );
 ```
+
+### Custom Headers
+
+If you need to pass headers to the proxied request, you can than add those headers during the configuration of the proxy.
+
+```javascript
+// server/index.js
+import koa from 'koa';
+import session from 'koa-session';
+import proxy, {ApiVersion} from '@shopify/koa-shopify-graphql-proxy';
+
+const app = koa();
```

Comment:
I know the other examples have this lowercase, but this should technically be `app = new Koa()`, no?